### PR TITLE
CONSOLE-2772: Consolidate A-D i18n namespaces

### PR DESCRIPTION
**Pull request by zherman0**
[CONSOLE-2772](https://issues.redhat.com/browse/CONSOLE-2772)
Consolidate A-E i18n namespaces

### CODE REVIEW

**Comment by spadgett** [comment]
Thanks @zherman0. @rebeccaalpert Do you know if this will conflict with #8101?

**Review by spadgett — APPROVED** [review]
/lgtm
/hold

I'm debating whether we should try to manually move the translations into their respective public catalogs or just wait for the next translation drop. It might be something we can skip @rebeccaalpert Any opinion?

**Comment by rebeccaalpert** [comment]
@spadgett - I think there will be conflicts. They should be easy to resolve though.

For me, it comes down to how much you value developer time. I think it's a pain to have to manually update the languages; we don't have any tooling around that the way we do for English. 

We just sent over files for translation on Monday, so the next drop won't include these changes. We'll have bad data for at least another two weeks if we don't manually update these, but the translation import process is quick. I'd lean towards waiting for new translation files if there's no pressing reason to have translations up to date.

**Comment by spadgett** [comment]
Let's hold for #8101 so that we don't have any problems backporting the updated translations.

/hold

**Comment by spadgett** [comment]
Hey, @zherman0. This need will rebase, then we can ask QE to review. Thanks!

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/assign @yapei @ahardin-rh @sferich888 
for approvals

See the note here: https://github.com/openshift/console/pull/8280#pullrequestreview-615749461

**Comment by yapei** [comment]
@XiyunZhao  Please help test this PR

**Comment by spadgett** [comment]
/lgtm cancel

I noticed that some of messages in packages use the public namespace. Those packages should use the namespace that matches the package name.

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/breakdown-body.tsx
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/breakdown-body.tsx:31]: Messages in ceph-storage-plugin should use the `ceph-storage-plugin` namespace.

```suggestion
        {t('ceph-storage-plugin~Not available')}
```

File: frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx
Comment (spadgett) [inline frontend/packages/console-shared/src/components/dashboard/activity-card/ActivityBody.tsx:24]: Use `console-shared` throughout this package.

```suggestion
          {t('console-shared~Started')}{' '}
```

File: frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.tsx
Comment (spadgett) [inline frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.tsx:49]:
```suggestion
          ariaChartLinkLabel={t('dev-console~View metrics for {{title}}', {
```

File: frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.spec.tsx
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.spec.tsx:541]:
```suggestion
      'operator-lifecycle-manager~Details',
```

File: frontend/packages/pipelines-plugin/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx
Comment (spadgett) [inline frontend/packages/pipelines-plugin/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx:16]: This should be `pipelines-plugin`

**Review by spadgett — CHANGES_REQUESTED** [review]
@zherman0 Sorry for missing this on previous review. I thought all of these files were under the `public` folder. For other packages, the namespace should match the package name.

**Comment by zherman0** [comment]
@spadgett - so the 'dashboard' namespace was used across multiple plugins.  I converted every plugin to its own namespace now.

**Comment by XiyunZhao** [comment]
Currently didn't find any issue on private build, @zherman0 could you please help to resolve the conflict, and i would like to double check on official build and cluster

**Comment by zherman0** [comment]
@spadgett - Rebased and ready for review.  Thanks.

**Comment by spadgett** [comment]
@cloudbehl We might need your help understanding the ceph test failures in this PR. Thanks!

File: frontend/packages/dev-console/locales/en/dev-console.json
Comment (spadgett) [inline frontend/packages/dev-console/locales/en/dev-console.json:2]: Something isn't right here... We should make sure we're using the same namespace as the rest of dev console. We could consider removing this from your PR since it's really code owned by the dev console team.

File: frontend/packages/ceph-storage-plugin/locales/en/dashboard.json
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/locales/en/dashboard.json:2]: I'm unsure why we have a new namespace in ceph-storage-plugin. @zherman0 Was this created on purpose?

Comment (zherman0) [reply frontend/packages/ceph-storage-plugin/locales/en/dashboard.json:2]: It was on purpose... once I removed the dashboard.json from public/locales/en, the i18n auto generated that file to the /package/ceph/locales.  The complete solution would be to go find where that is used in the ceph code and change it to ceph-storage.json but I decided to leave that out of this scope since it involves 5 ceph files.  Let me know if you want me to make those changes as part of this PR.

Comment (spadgett) [reply frontend/packages/ceph-storage-plugin/locales/en/dashboard.json:2]: @zherman0 Can you open a bug against the storage plugin component to make this change?

**Review by spadgett — APPROVED** [review]
/lgtm

We will need docs and px approval

File: frontend/packages/ceph-storage-plugin/locales/en/dashboard.json
Comment (zherman0) [reply frontend/packages/ceph-storage-plugin/locales/en/dashboard.json:2]: This has been resolved by PR 8764.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
